content/en/_index.md

```diff
@@ -378,6 +462,20 @@ try (CloudClient dlp = CloudClient.create()) {
   // make a request with the client
 }
 {{< /tab >}}
+{{< tab header="Go" >}}
+// Initialize client that will be used to send requests. This client only needs
+// to be created once, and can be reused for multiple requests. After completing
+// all of your requests, call the "Close" method on the client to safely clean
+// up any remaining background resources.
+ctx := context.Background()
+c, err := foo.NewClient(ctx)
+if err != nil {
+    // TODO(developer): handle error
```

Comment:
I think we should update these TODOs to match the guidance on writing out output as well

Reply:
Done.